src/pcre2_jit_neon_inc.h

```diff
@@ -124,7 +124,7 @@ vect_t vmask = VDUPQ(mask);
 compare_type compare1_type = compare_match1;
 compare_type compare2_type = compare_match1;
 vect_t cmp1a, cmp1b, cmp2a, cmp2b;
-const sljit_u32 diff = IN_UCHARS(offs1 - offs2);
+const sljit_uw diff = IN_UCHARS(offs1 - offs2);
```

Comment:
@zherczeg I just wanted to point out this small change in the NEON code. I think it's correct.

Reply:
Likely. diff is a value between 0 and 31